students/douglas_klos/session9/mailroom/donor.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+#pylint: disable=R1710, C0321, W1113
+""" Mailroom OO Donor class """
+
+# Douglas Klos
+# March 14th, 2019
+# Python 210, Session 9, Mailroom OO
+# donor.py
+
+import datetime
+
+
+class Donor():
+    """
+    Donor class
+
+    Attributes:
+        name : The name of the donor
+        donations : The donations for the donor
+    """
+
+    THANK_YOU_LETTER = ('Dear {}:\n'
```

Comment:
This constant probably belongs as part of the user interface. Only Donor information should be here.

Reply:
Of course, the argument could be made that you want the same thank you letter displayed, regardless of what the user interface is.